test/rhsmlib/dbus/test_register.py

```diff
@@ -413,6 +421,26 @@ def test_Register(self):
         result = self.obj.Register.__wrapped__(self.obj, "org", "username", "password", {}, {}, self.LOCALE)
         self.assertEqual(expected, json.loads(result))
 
+    def test_RegisterWithForceOption(self):
```

Comment:
Please do not use camel case in method names. Python style coding uses `different_pattern_of_class_names`. :-) Your another test (`test_Register__already_registered`) uses good pattern.